src/main/java/com/thebois/models/beings/Colony.java

```diff
@@ -45,8 +45,8 @@ public Colony(final Iterable<Position> vacantPositions, final IPathFinder pathFi
         setBeings(pawns);
     }
 
-    public IInventory getInventory() {
-        return inventory;
+    public IInventory getColonyInventory() {
```

Comment:
style

Now you've made it into `colony.getColonyInventory()`, it should just be `colony.getInventory()`, no need to repeat the word "Colony" everywhere.